src/store/getters.js

```diff
@@ -2,4 +2,7 @@ export default {
   autocompleteTags(state) {
     return state.tags.map(tag => ({ label: tag.name, value: tag.slug }));
   },
+  isLoggedIn(state) {
```

Comment:
Burada bu getter'in cok bir manasi yok. 

Vuex'de `getter`'lar genelde var olan data'yi alip degistirip geri donmek icin kullanilir. Yukardaki `autocompleteTags`'e bakin mesela. `state.tags`'i aliyor ve `map` edip yeni bir array donuyor. Biz burada zaten `state`'de var olan bir seyi hic degistirmeden kullaniyoruz.

Bunu kullandigin component'de `...mapGetters(['isLoggedIn'])` kullandin ya, bunu silip yerine `...mapState(['isLoggedIn'])` kullanirsan da ayni sey olacaktir.